test/jsxFactory.tsx

```diff
@@ -0,0 +1,83 @@
+import * as React from 'react';
```

Comment:
Test cases have been almost rewritten in a DOM-free way (using `enzyme`'s shallow rendering).
The next one, however, was difficult to rewrite with my limited knowledge.

https://github.com/cyclejs/react-dom/blob/229e592ab80998ace5bf4fd0c12eee36f4499731/test/jsx-factory.tsx#L152-L185

Any good ideas?